packages/gatsby/src/schema/print.ts

```diff
@@ -342,20 +343,13 @@ export const printTypeDefinitions = ({
   }
 
   const internalTypes = [
-    `Boolean`,
+    ...builtInScalarTypeNames,
     `Buffer`,
```

Comment:
In this PR I essentially just dropped `Json` from the list, but we still have some questionable type names here - in particular `Buffer` is raising my eye brow. For now leaving it as I don't quite understand the reason for it + it doesn't seem to cause problems (yet)